mainnet-icos-revisions.json

```diff
@@ -4,25 +4,485 @@
       "tdb26-jop6k-aogll-7ltgs-eruif-6kk7m-qpktf-gdiqx-mxtrf-vb5e6-eqe": {
         "version": "a299620cd5e73ed21b415f2ebe3da0bb43a3649d",
         "update_img_hash": "63a8dc2707fead820fb7971d5986a17e8e67074b0493198c796c5d799c5a7a13",
-        "update_img_hash_dev": "25077e209a5b876ca13c93710277a5a4689fece32538b355bab2651231c5388e"
+        "update_img_hash_dev": "25077e209a5b876ca13c93710277a5a4689fece32538b355bab2651231c5388e",
+        "launch_measurements": {
+          "guest_launch_measurements": [
+            {
+              "measurement": [
```

Comment:
IMHO a base64 encoded measurement would be more compact and easier to read. But up to you.

Reply:
This takes the same format as [proposals](https://dashboard.internetcomputer.org/proposal/138476) and [`launch-measurements.json`](https://download.dfinity.systems/ic/a299620cd5e73ed21b415f2ebe3da0bb43a3649d/guest-os/update-img/launch-measurements.json), so I will leave it for now, but we may want to update everywhere to some more compact format in the future.

CC: @frankdavid